diff -urN eog-plugins-2.28.1/configure.ac eog-plugins-2.28.1.new/configure.ac
--- eog-plugins-2.28.1/configure.ac	2009-09-27 11:18:49.000000000 +0200
+++ eog-plugins-2.28.1.new/configure.ac	2009-11-13 10:14:16.187350297 +0100
@@ -192,11 +192,11 @@
 	AC_MSG_RESULT([$PYGTK_DEFSDIR])
 
 	AC_MSG_CHECKING([for pygtk codegen])
-	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.py"
+	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.pyc"
 	AC_MSG_RESULT([$PYGTK_CODEGEN])
 
 	AC_MSG_CHECKING([for pygtk h2def])
-	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.py"
+	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.pyc"
 	AC_MSG_RESULT([$PYGTK_H2DEF])
 
 	AC_SUBST([PYGTK_DEFSDIR])
@@ -280,14 +280,6 @@
 # Postr 
 # *****
 plugin_defined postr
-if test "$?" = 1
-then
-	AC_PATH_PROG(POSTR, postr)
-	if test -z "$POSTR"
-	then
-		undef_plugin postr "postr was not found"
-	fi
-fi
 
 # *********
 # Champlain 
@@ -306,8 +306,8 @@
 then
 	
 	PKG_CHECK_MODULES(CHAMPLAIN,
-		[ champlain-0.4 >= 0.4,
-		  champlain-gtk-0.4 >= 0.4,
+		[ champlain-0.6 >= 0.6,
+		  champlain-gtk-0.6 >= 0.6,
 		  clutter-gtk-0.10 >= 0.10,
 		  libexif >= 0.6.16 ],
 		[],[have_libchamplain=no])
